### DISCUSSION TITLE
Some advice about customer management.

### DISCUSSION THREAD

**Discussion by offlink-io** [swellstores/community#473] [Help] [answered]
Hello,

I need some advice about customer management.

I'm currently developing a swell-based solution for product and customer management, ....

In this solution, I have several organizations that register and I would like to segment the customers according to each organization (ex: orga1 to customer1, customer2 | orga2 to customer3,customer4, ...)

The idea here is that a customer can, for example, call or e-mail the organization to order a product sold by that organization.

Tomorrow, I'd like to add a new acquisition channel via a marketplace that will enable customers to order products directly from organizations without having to do it manually.

However, potentially a customer1 can order a product from organization1 and also from organization2.

My question is how can we segment this correctly so that organization1 can find all the orders and details (address, etc.) of customer1 without seeing the data linked to organization2 because they had the same customer1?

What do you recommend?

Create a group for each organization, then link users to each group (https://developers.swell.is/guides/core-concepts/customers#customer-groups).

Add custom fields to certain templates?

Thanks for your help!

Translated with DeepL.com (free version)

**Comment by sags95** [comment, marked as answer]
You could certainly use customer groups, another way could be to extend the /accounts model and include an 'organization' lookup field. This would allow you to relate one account to another (either at time of creation or later in the customer journey) by updating the customer record. 

https://developers.swell.is/guides/custom-fields-and-models
